contracts/Lens/PoolLens.sol

```diff
@@ -112,9 +112,9 @@ contract PoolLens is ExponentialNoError {
         // The market's last updated rewardTokenBorrowIndex or rewardTokenSupplyIndex
         uint224 index;
         // The block number the index was last updated at
-        uint32 block;
+        uint256 block;
         // The block number at which to stop rewards
-        uint32 lastRewardingBlock;
+        uint256 lastRewardingBlock;
```

Comment:
To avoid confusions I think we should keep consistent naming of variables.

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/324/commits/aca35e56df275728c3ee83948cfe9164f049776e